bundles/action/org.openhab.action.astro/src/main/java/org/openhab/action/astro/internal/AstroActionService.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.action.astro.internal;
+
+import java.util.Dictionary;
+
+import org.openhab.core.scriptengine.action.ActionService;
+import org.osgi.service.cm.ConfigurationException;
+import org.osgi.service.cm.ManagedService;
+
+/**
+ * This class registers an OSGi service for the Astro action.
+ * 
+ * @author Gerhard Riegler
+ * @since 1.7.0
+ */
+public class AstroActionService implements ActionService, ManagedService {
+
+	public void activate() {
+	}
+
+	public void deactivate() {
+	}
+
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public String getActionClassName() {
+		return Astro.class.getCanonicalName();
+	}
+
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public Class<?> getActionClass() {
+		return Astro.class;
+	}
+
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public void updated(Dictionary<String, ?> properties) throws ConfigurationException {
+	}
```

Comment:
since updated() isn't implemented in your Action doesn't need to implement ManagedService